packages/desktop/webpack/webpack.config.renderer.dev.js

```diff
@@ -17,7 +17,8 @@ module.exports = {
     extensions: ['.ts', '.tsx', '.js', '.jsx'],
     plugins: [
       new TsconfigPathsPlugin({ configFile: 'tsconfig.build.json' })
-    ]
+    ],
+    conditionNames: ["import", "node"]
```

Comment:
Changed due to issues with @ipld/dag-cbor. Moving the logic from state-manager to the backend in a future PR will make these changes unnecessary.